mobile/lib/common/channel_status_notifier.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:f_logs/f_logs.dart';
+import 'package:flutter/material.dart';
+import 'package:get_10101/bridge_generated/bridge_definitions.dart' as bridge;
+import 'package:get_10101/common/application/event_service.dart';
+import 'package:get_10101/ffi.dart';
+
+class ChannelStatusNotifier extends ChangeNotifier implements Subscriber {
+  bridge.ChannelStatus? latest;
+
+  ChannelStatusNotifier();
+
+  bridge.ChannelStatus? getChannelStatus() {
```

Comment:
🤔 it might be nicer if we didn't need to return a nullable type here (see other discussion topic )